src/vs/workbench/parts/files/electron-browser/views/explorerViewer.ts

```diff
@@ -296,6 +296,7 @@ export class FileRenderer implements IRenderer {
 			}, 0);
 		});
 
+		const initialRelPath: string = relative(stat.root.resource.fsPath, stat.parent.resource.fsPath);
```

Comment:
You only use this const further down. It is usually best to intiilize things exactly where you use them so they are scoped in the right place, please move it to the place where you actually use it.